build/emtests.rs

```diff
@@ -4,41 +4,43 @@
 //! - Compile using emcc and get the .wasm from it (wasm)
 //! - Generate the test that will compare the output of running the .wasm file
 //!   with wasmer with the expected output
+use glob::glob;
 use std::fs;
+use std::path::Path;
 use std::path::PathBuf;
 use std::process::Command;
 
+use std::fs::File;
+use std::io::prelude::*;
+use std::io::BufReader;
+
 static BANNER: &str = "// Rust test file autogenerated with cargo build (build/emtests.rs).
 // Please do NOT modify it by hand, as it will be reseted on next build.\n";
 
-const TESTS: [&str; 4] = [
-    "emtests/env.c",
-    "emtests/puts.c",
-    "emtests/printf.c",
-    "emtests/clock_gettime.c",
-];
+const EXTENSIONS: [&str; 2] = ["c", "cpp"];
+const EXCLUDES: [&str; 1] = ["localtime.c"];
 
-pub fn compile(file: &str) -> String {
+pub fn compile(file: &str, ignores: &Vec<String>) -> Option<String> {
     let mut output_path = PathBuf::from(file);
     output_path.set_extension("out");
-    let output_str = output_path.to_str().unwrap();
+    //    let output_str = output_path.to_str().unwrap();
 
     // Compile to .out
-    Command::new("cc")
-        .arg(file)
-        .arg("-o")
-        .arg(output_str)
-        .output()
-        .expect("failed to execute process");
+    //    Command::new("cc")
+    //        .arg(file)
+    //        .arg("-o")
+    //        .arg(output_str)
+    //        .output()
+    //        .expect("failed to execute process");
 
     // Get the result of .out
-    let output = Command::new(output_str)
-        .arg(output_str)
-        .output()
-        .expect("failed to execute process");
+    //    let output = Command::new(output_str)
+    //        .arg(output_str)
+    //        .output()
+    //        .expect("failed to execute process");
 
     // Remove executable
-    fs::remove_file(output_str).unwrap();
+    //    fs::remove_file(output_str).unwrap();
```

Comment:
If we no longer run `cc`, why not do the same with the `emcc`?

Reply:
The emscripten tests don't include generated wasm files like they do the output. Also, we may need to updated the wasm files as EMCC is updated.  If you're just want to regenerate test files though it much faster to comment out EMCC temporarily. Maybe it should be commented out by default.